src/umzug.ts

```diff
@@ -327,15 +327,15 @@ export class Umzug<Ctx> extends EventEmitter {
 
 		for (const m of toBeReverted) {
 			const start = Date.now();
-			this.logging('== ' + m.name + ': reverting =======');
+			this.logging({ event: 'reverting', name: m.name });
 			this.emit('reverting', m.name, m);
 
 			await m.down?.({ name: m.name, path: m.path, context: this.options.context });
 
 			await this.storage.unlogMigration(m.name);
 
-			const duration = ((Date.now() - start) / 1000).toFixed(3);
-			this.logging(`== ${m.name}: reverted (${duration}s)\n`);
+			const duration = Number.parseFloat(((Date.now() - start) / 1000).toFixed(3));
```

Comment:
```suggestion
			const duration = (Date.now() - start) / 1000;
```

This makes it equivalent to what was done a few lines above. Also I think dividing an integer by 1000 can never introduce floating point errors, so my suggestion is probably strictly equivalent to the current behavior anyway.

Reply:
Ended up doing this in a later PR: https://github.com/sequelize/umzug/pull/397/files#diff-98ea65505aaa84092a04a27b8479d32354660e2d23b914f4e44287f9745a8de6R312